.github/workflows/test.yml

```
@@ -41,20 +79,17 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-cargo-${{ matrix.rust }}-
           ${{ runner.os }}-cargo-
-    
+
     - name: Build
-      shell: bash
-      run: |
-          cargo build --verbose --no-default-features
-    
+      shell: ${{ matrix.os == 'windows-latest' && 'msys2 {0}' || 'bash' }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)